homeassistant/components/lifx/__init__.py

```diff
@@ -22,39 +45,173 @@
     }
 )
 
-CONFIG_SCHEMA = vol.Schema(
-    {DOMAIN: {LIGHT_DOMAIN: vol.Schema(vol.All(cv.ensure_list, [INTERFACE_SCHEMA]))}},
-    extra=vol.ALLOW_EXTRA,
+CONFIG_SCHEMA = vol.All(
+    cv.deprecated(DOMAIN),
+    vol.Schema(
+        {
+            DOMAIN: {
+                LIGHT_DOMAIN: vol.Schema(vol.All(cv.ensure_list, [INTERFACE_SCHEMA]))
+            }
+        },
+        extra=vol.ALLOW_EXTRA,
+    ),
 )
 
 DATA_LIFX_MANAGER = "lifx_manager"
 
 PLATFORMS = [Platform.LIGHT]
+DISCOVERY_INTERVAL = timedelta(minutes=15)
+MIGRATION_INTERVAL = timedelta(minutes=5)
+
+DISCOVERY_COOLDOWN = 5
+_LOGGER = logging.getLogger(__name__)
```

Comment:
(Maybe in a future PR) This can go as a single `LOGGER` in `const.py` (which will free up each platform from having to define its own).

Reply:
c90f5e1f34c8dffde5437f81e800c01f9b8dd714